src/App.js

```diff
@@ -59,25 +55,22 @@ function App() {
     setGuesses(0)
   }
 
-  let colors = swatches.map((swatch) => {
-    return(
+  let colors = swatches.map((swatch) => (
```

Comment:
I guess the return here is implied?

Reply:
Exactly, on a map function, if your not doing any logic inside your callback, the return is not needed. But this is not a written rule, I just like it better this way, at the end, is up to you